tests/testthat/test-derive.R

```diff
@@ -16,6 +18,26 @@ with_test_authentication({
         expect_identical(as.vector(ds$v3a), as.vector(ds$v3) + 5)
     })
 
+    test_that("derivation pulls the derivation", {
+        expect_is(derivation(ds$v3a), "CrunchExpr")
+        expect_fixed_output(derivation(ds$v3a), "Crunch expression: v3 + 5")
+    })
+
+    test_that("derivation returns NULL if the variable is not derived", {
+        expect_null(derivation(ds$v3))
+    })
+
+    test_that("derivation()<- input validation ", {
+        expect_error(derivation(ds$v3) <- ds$v3 + 1,
+                     paste0("The variable ", dQuote("v3"),
+                            " must already be a derived variable."))
+        try(ds$v3c <- ds$v3 + 5)
```

Comment:
Please. It was to prevent tests from exiting prematurely on error but not fail the Jenkins build, but that's been handled elsewhere now.